src/types/types.ts

```diff
@@ -1031,6 +1031,7 @@ export interface EdgeCurrencyTools {
   readonly getDisplayPublicKey?: (
     publicWalletInfo: EdgeWalletInfo
   ) => Promise<string>
+  readonly getMasterPublicKeys?: () => { [key: string]: string }
```

Comment:
You need to pass the private / public keys in as parameters. The tools have no state.

Since this is meant for the user to view, not for the app to read, `getDisplayPublicKeys` might be a better name?

The `derivePublicKey` gives the machine-readable format, and the `getDisplayPublicKey` returns a single string, with the downside that this isn't broken down and copy-pasteable. So we want a name that fits with the scheme.